src/components/overlays/pause.js

```diff
@@ -20,9 +20,20 @@ export class Pause extends Screen {
 
         const parentKey = this.context.parentScreens.slice(-1)[0].scene.key;
         const showReplay = Boolean(this.context.navigation[parentKey].routes.restart);
+        const showSelect = Boolean(this.context.navigation["pause"].routes.select);
```

Comment:
doesn't really matter, but you can just do `.navigation.pause.routes` instead of the `["pause"]` on these